server/conf/cms/secrets.sample.py

```diff
@@ -154,19 +153,25 @@
     - Any new selectors or css styles (add to /taccsite_cms/static/site_cms/css/src/_imports/branding_logos.css)
     - Image files being references (add to /taccsite_cms/static/site_cms/img/org_logos)
 
-Values to populate:
-
-_SETTING_NAME = [                 # The name of the branding or logo config setting object.
-    "org_name",                         # The name of the organization the branding belongs too.
-    "img_file_src",                      # Path and filename relative to the static files folder.
-    "img_element_classes",        # The list of selectors to apply to the rendered element, these need to exist in the css/scss.
-    "a_target_url",                      # The href link to follow when clicked, use "/" for portal logos.
-    "a_target_type",                    # The target to open the new link in, use _blank for external links, _self for internal links.
-    "alt_text",                             # The text to read or render for web assistance standards.
-    "cors_setting",                      # The CORS setting for the image, set to anonymous by default.
-    "visibility"                             # Toggles wether or not to display the element in the template, use True to render, False to hide.
+Values to populate (for an array):
+
+_SETTING_NAME = [                # The name of the branding or logo config setting object.
+    "org_name",                    # The name of the organization the branding belongs too.
+    "img_file_src",                # Path and filename relative to the static files folder.
+    "img_element_classes",         # The list of selectors to apply to the rendered element, these need to exist in the css/scss.
+    "a_target_url",                # The href link to follow when clicked, use "/" for portal logos.
+    "a_target_type",               # The target to open the new link in, use _blank for external links, _self for internal links.
+    "alt_text",                    # The text to read or render for web assistance standards.
+    "cors_setting",                # The CORS setting for the image, set to anonymous by default.
+    "visibility"                   # Toggles wether or not to display the element in the template, use True to render, False to hide.
 ]
 
```

Comment:
How do these _SETTING_NAME values work together? Won't the second instance simply clobber the first so that it always resolves to `_SETTING_NAME= { "img_file_src": "..." }`?

Reply:
The `_SETTING_NAME` is a placeholder for the name of a setting, like `_ANORG_BRANDING` or `_ANORG_LOGO`. You authored this in the CMS. See [TACC/Core-CMS: 10b82cb "Added a secrets_example.py file to get things runnning out of the box."](https://github.com/TACC/Core-CMS/commit/10b82cb08a5ff377506982c44469ee437fb8a9da#diff-06639010e885235ef95c8c01d5e227691057be093c3246f02c522f639c911e1dR31-R66)

Reply:
If you're asking specifically about the object ones, I believe they also won't be clobbering each other, because the `_SETTING_NAME` is unique.

- I added documentation and instance of an object-based setting in the CMS. See [TACC/Core-CMS: c8844e1 "FP-947: Support Favicon as Custom Asset"](https://github.com/TACC/Core-CMS/commit/c8844e13090196e1e1e91c098acaf4fde64081f8).
- It is based off of the object setting I made for Portal nav. See [TACC/Core-CMS: 5689bca "GL-29: Generic customizable auth and unath menus"](https://github.com/TACC/Core-CMS/commit/5689bca48740a098ab81f401195ce6f68a390754).

Reply:
But these are not unique names:

_SETTING_NAME = []

_SETTING_NAME = {}

Python will overwrite the array values (above) with the dict values (below).
There will only ever be one setting named `_SETTING_NAME`.
I just don't grok the intended use case of the dict vs the array.

Reply:
This is in a big docstring block fyi, from line 143-206

Reply:
The use case is dict. The array is a first attempt for complex settings. The arrays should become objects when we have time to convert them.

Reply:
Ahhh, did not see the docstring beginning way up there, love the way github presents diffs...